ui/apps/nft-portal/src/app/components/transactions/transaction-content.tsx

```diff
@@ -30,6 +45,180 @@ export function TransactionContent({ identifier, transaction, network, vault }:
         setup()
     }, [identifier, transaction, vault])
 
+    useEffect(() => {
+        setInteractionTemplateData(null)
+        setInteractionTemplateAudits(null)
+    }, [transactionData])
+
+    useEffect(() => {
+        async function getInteractionTemplateAudits() {
+            if (interactionTemplateData) {
+                const auditors = await fetch(`https://flix.flow.com/v1/auditors?network=${network}`)
```

Comment:
I'm guessing this data is relatively static? Does the FLIX API serve the necessary `Cache-Control` logic to allow the browser to cache this locally?

Reply:
Yea this endpoint is fairly static! The API also sits behind a cache. I'll look into the FLIX API to be sure we're setting the correct headers!